ntbs-service/Pages/Shared/_Header.cshtml

```diff
@@ -42,7 +42,8 @@
                         @environmentDescription.DisplayName
                     </strong>
                     <span class="govuk-phase-banner__text">
-                        The data on this site is synthetic and is for @environmentDescription.DisplayName purposes only.
+                        The data on this site is @(environmentDescription.ContainsLiveData ? "a copy of live" : "synthetic")
```

Comment:
Maybe "a copy of the live data" would be slightly clearer?

Reply:
Yeah I agree, I've change it to that